lib/search.rb

```diff
@@ -7,11 +7,23 @@ def self.create(environment = Rails.env)
     config_path = Rails.root + 'config' + "elasticsearch.yml"
     client_config = HashWithIndifferentAccess.new(YAML.load_file(config_path))
     client_config = client_config[environment].merge(client_config[:all_envs])
+    index_name = client_config.delete(:index)
```

Comment:
I thought so too, but didn't want to change it too much to avoid a massive diff. I tried to keep this to the point, but you're right in that this is quite confusing. I can add a new commit to refactor this once we're happy with the new library. Does that make sense?